### ISSUE TITLE
[feature suggestion] new options

### ISSUE THREAD

**Issue by AlienLifeForm1** [Brummolix/AutoarchiveReloaded#48]
It would be nice to have this options:

     - delay: time in minutes after startup, before autoarchivereloaded begins to work
     - interval: in days, so that I can say only archive every 14 days
     - popup: to ask when delay and interval saying it's time to archive, that archive starts only after confirming the popup

best regards,
Alf

**Comment by Brummolix** [comment]
Make sense to me... I will check this
